services/installer.sh

```
@@ -30,23 +30,38 @@ if ! getent passwd keylime >/dev/null; then
             keylime
 fi
 
-echo "Changing files to be owned by the keylime user"
-# Create all directories required if not there
-mkdir -p /var/lib/keylime
-mkdir -p /var/log/keylime
-mkdir -p /var/run/keylime
+# install TPM certificate store to /usr/share/keylime/
+# tmpfiles.d will copy this to /var/lib/keylime/tpm_cert_store
+mkdir -p /usr/share/keylime
+cp -r $BASEDIR/../tpm_cert_store /usr/share/keylime/
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)